text/0000-temp-components-lib.md

```diff
@@ -0,0 +1,622 @@
+with the Pando Normalize/Font setup [mentioned in the docs](https://design.pluralsight.com/docs/next/learn/get-started/installation/add-to-website). Users will _not_ need to install any other
+Pando libraries to use the React one because it will use them internally.
+
+### Is this a breaking change?
+
+By nature of introducing a completely new design, yes, it will be a breaking change
+but easier for most users to migrate than using Headless-styles due to having an
+API that is familair.
+
+### Can we write a codemod?
+
+Could be worth exploring for _Classic users_. Classic by nature is not 1:1 to Pando
+(and never will be) and is quite honestly very inconsistent in how it designs
+components. However, it might work for the components that are used in the Pando
+UI Kit.
+
+There would need to be the following features:
+
+1. Correct the import paths in each file
+2. Correct the component usage and it's props in each file - without causing bugs
+3. Delete unused Classic component code that is naturally built into Pando (there is a lot)
+4. Delete unused Classic dependencies (see [migration list](https://design.pluralsight.com/docs/next/learn/learn-pando/migration)) from the users package.json
+  a. Recognize whether a project uses Yarn/NPM and re-install everything automatically
+5. Log warnings/errors for old Classic components that are no longer supported but are still around
+  a. Probably a nice interface like "Pando no longer uses the Breadcrumb component, would you like to keep it? (Y/N)"
+
+:warning: This should definitely be approached with caution! :warning:
+
+If we can make this happen, the ROI on business **would be unlike anything
+Pluralsight as a company has ever seen** regarding migrating to a new brand/tooling.
+
+### Should we coordinate with other libraries within Pando?
+
+Yes. All Pando libraries (i.e. and public package) should share the same version
+number to make using and maintaining each package easier for the developer experience.
+
+This should make it easier for a user to know if they are using compatible versions
+if they choose to use multiple libraries within a single project (i.e. using both
+`react` and `headless-styles`).
+
+## How would we teach this?
+
+The naming and terminology should be a direct mirror of the Headless-styles libray
+which in turn mirrors the Design-Tokens. This helps create a consistent eco-system
+that will scale throuhout time.
+
+This will be presented in the Docs > Reference > [Component Name] page which would
+replace the Headless-styles content with the component design. This update
+would create the following change to the "story" of each reference page:
+
+1. Intro
+2. Anatomy
+3. Import (component & types)
+4. Usage
+  a. Basic (component)
+  b. Variants* (component)
+5. Extending
+  a. Classes (about using unused classes shipped with Headless-styles)
+  b. Ejecting (using Headless-styles)
+6. Reference (component)
+  a. Props
+  b. Caveats
```

Comment:
minor: I think these might not be rendering correctly because they need to be indented over 4 spaces instead of 2? 😅 

```suggestion
4. Usage
    a. Basic (component)
    b. Variants* (component)
5. Extending
    a. Classes (about using unused classes shipped with Headless-styles)
    b. Ejecting (using Headless-styles)
6. Reference (component)
    a. Props
    b. Caveats
```